docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+- Outlook
+- PowerPoint
+- Word
+
+The [5-minute quick start](../index.md) for each of these four applications contains step-by-step instructions that describe how to create an Office Add-in using Visual Studio. Within a quick start, select the `Visual Studio` tab to display the proper set of instructions, as shown in the following screenshot.
+
+![A screenshot that shows quick start instructions with the Visual Studio tab selected](../images/quick-start-tab-vs.png)
+
+## Exploring APIs with Script Lab
+
+Script Lab is an add-in that enables you to explore the Office JavaScript API and run code snippets while you're working in an Office program such as Excel or Word. It's available for free via [AppSource](https://appsource.microsoft.com/product/office/WA104380862) and is a useful tool to include in your development toolkit as you prototype and verify the functionality you want in your add-in. In Script Lab, you can access a library of built-in samples to quickly try out APIs or even use a sample as the starting point for your own code. 
+
+For more information about Script Lab, see [Explore Office JavaScript APIs using Script Lab](../overview/explore-with-script-lab.md).
+
+## Next steps
+
+This article has described important Office Add-ins development concepts, outlined the different ways to create Office Add-ins, and introduced Script Lab as a valuable tool for exploring Office JavaScript APIs and prototyping add-in functionality. Now that you've explored this introductory information, consider continuing your Office Add-ins journey as described in the remainder of this section.
+
+### Create an Office add-in
+
+You can quickly create a very basic add-in for Excel, OneNote, Outlook, PowerPoint, Project, or Word by completing a [5-minute quick start](../index.md). If you've previously completed a quick start and want to create a slightly more complex add-in, you can do so by completing a [tutorial](../index.md).
+
+### Explore the APIs with Script Lab
+
+Explore the library of built-in samples in [Script Lab](../overview/explore-with-script-lab.md) to get a sense for the capabilities of the Office JavaScript APIs.
+
+### Learn more about developing, testing, and publishing Office Add-ins
+
+> [!IMPORTANT]
+> The table of contents (TOC) within this documentation is organized as follows:
+> 
+> - Articles that apply to multiple types of Office Add-ins (i.e., Excel, Word, Outlook, etc.) appear under [Office Add-ins guidance](../overview/index.md) in the TOC.
+> 
+> - Articles that apply to only one type of Office Add-in appear within the corresponding host-specific section of the TOC (for example, [Excel add-ins](../excel/index.md)).
+> 
+> For any type of add-in that you build, you'll use information in the [Office Add-ins guidance](../overview/index.md) section of this documentation, along with information in the host-specific section that corresponds to the type of add-in you're building (for example, [Excel add-ins](../excel/index.md)).
+
```

Comment:
Should we mention ref doc versioning in this TOC explanation (or ref docs at all)?